docs/src/components/pricing/PricingTable.tsx

```diff
@@ -79,84 +86,132 @@ interface PlanPriceProps {
 export function PlanPrice(props: PlanPriceProps) {
   const { plan } = props;
 
+  const { licenseType } = useLicenseType();
+  const annual = licenseType === 'Annual';
+  const planPriceMinHeight = 64;
+
   if (plan === 'community') {
     return (
-      <Box sx={{ display: 'flex', alignItems: 'center', mt: 1 }}>
-        <Typography variant="h4" component="div" fontWeight="bold" color="success.600">
-          $0
-        </Typography>
-        <Box sx={{ width: 5 }} />
-        <Typography variant="body2" color="text.secondary" sx={{ mt: '4px' }}>
-          – free forever
+      <React.Fragment>
+        <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center', mt: 1, mb: 4 }}>
+          <Typography
+            variant="h3"
+            component="div"
+            fontWeight="bold"
+            color="success.600"
+            sx={{ mt: 4.5 }}
+          >
+            $0
+          </Typography>
+        </Box>
+        <Typography variant="body2" color="text.secondary" textAlign="center">
+          Free forever!
         </Typography>
-      </Box>
+      </React.Fragment>
     );
   }
+
+  const monthlyDisplay = annual;
+
+  const priceUnit = monthlyDisplay ? '/ dev / month' : '/ dev';
```

Comment:
Would it make more sense to get rid of the monthly price?
After switching between Annual/Perpetual tabs, the first thing I did was calculate the price for Annual per year (15 * 12) to be able to compare it with Perpetual.
I didn't even notice "Billed annually at $180/dev." note below 😅

https://user-images.githubusercontent.com/13808724/220967961-bfce0878-3011-48e0-8ad1-10252aafea30.mov

Reply:
> Would it make more sense to get rid of the monthly price?

Why not try to change the display but I don't think it should happen in this iteration. I think it's more important to measure the impact of the perpetual license with as few other variable changes as possible. Maybe this could have an important impact.